data/json/items/armor.json

```diff
@@ -4924,19 +4924,20 @@
         "color" : "blue",
         "covers" : ["TORSO", "ARMS", "LEGS"],
         "to_hit" : -3,
-        "storage" : 10,
+        "storage" : 26,
         "symbol" : "[",
         "description" : "A heavy, full-length wool coat.  Cumbersome, but warm and with deep pockets.",
         "price" : 18000,
         "material" : ["wool", "null"],
-        "volume" : 16,
+        "volume" : 32,
         "cutting" : 0,
         "warmth" : 50,
         "phase" : "solid",
-        "encumbrance" : 20,
+        "environmental_protection" : 1,
+        "encumbrance" : 25,
```

Comment:
At 12 encumbrance for just 8/8 at 90% coverage, it is a rather bad choice.
If fur duster is too weak, maybe it should get a noticeable buff so that it becomes worth crafting.